sys/linux/dev_kvm.txt

```diff
@@ -454,7 +462,21 @@ kvm_msi {
 	pad	array[const[0, int8], 12]
 }
 
-kvm_one_reg {
+kvm_one_reg [
+	arm64_core	kvm_one_reg_arm64[kvm_regs_arm64_core]
+	arm64_fp	kvm_one_reg_arm64[kvm_regs_arm64_fp]
+	arm64_ccsidr	kvm_one_reg_arm64[kvm_regs_arm64_ccsidr]
+	arm64_sys	kvm_one_reg_arm64[kvm_regs_arm64_sys]
+	arm64_extra	kvm_one_reg_arm64[kvm_regs_arm64_extra]
+	other		kvm_one_reg_other
+]
+
+type kvm_one_reg_arm64[FTYPE] {
+	id	flags[FTYPE, int64]
+	addr	ptr64[inout, int64]
+}
+
+kvm_one_reg_other {
 	id	int64
 	addr	int64
```

Comment:
Should this also be `ptr64[inout, int64]`?

Reply:
I wanted to preserve the existing behavior, but, yes, we could do that.